node/src/chain_spec/mod.rs

```diff
@@ -0,0 +1,2 @@
+//! Frequency chain configurations.
+pub mod frequency_local;
```

Comment:
@wilwade  following Acala and Moonbeam design pattern, as a first step I did basic refactoring and will add more PRs for #234  incrementally